pkg/asset/cluster/azure/azure.go

```diff
@@ -38,7 +39,42 @@ func PreTerraform(ctx context.Context, clusterID string, installConfig *installc
 	if err != nil {
 		return errors.Wrap(err, "failed to get session")
 	}
-	return tagResourceGroup(ctx, clusterID, installConfig, session)
+
+	if err := tagResourceGroup(ctx, clusterID, installConfig, session); err != nil {
+		return err
+	}
+
+	if len(installConfig.Config.Azure.VirtualNetwork) == 0 {
+		return nil
+	}
```

Comment:
similar to previous comment, would be better to move this check within `tagVNet` in case we later extend the `PreTerraform` func.

Reply:
done